pkg/templatestore/templatestore.go

```diff
@@ -54,10 +53,7 @@ func Read(name string) ([]byte, error) {
 		name += ".yaml"
 	}
 	for _, templatesDir := range paths {
-		filePath, err := securejoin.SecureJoin(templatesDir, name)
-		if err != nil {
-			return nil, err
-		}
+		filePath := filepath.Join(templatesDir, name)
```

Comment:
Needs a code comment to explain why SecureJoin is not needed

Reply:
Also `name` needs to be validated

Reply:
Not sure what that means? You want to drop `SecureJoin` when processing template:// URLs and instead check `templateName` here to make sure it doesn't include `..` as a substring?

Reply:
SecureJoin can remain there in processing URLs, but we still have to make sure that something isn’t calling Read() with an unexpected value.

GoDoc should also explain the expected value of `name`